includes/Modules/Analytics.php

```diff
@@ -391,6 +413,12 @@ protected function get_datapoint_definitions() {
 				'shareable' => Feature_Flags::enabled( 'dashboardSharing' ),
 			),
 		);
+
+		if ( Feature_Flags::enabled( 'ga4Reporting' ) ) {
+			unset( $datapoints['POST:create-account-ticket'] );
```

Comment:
I originally did that too, but did it this way to minimize the diff since it wasn't that expensive. In the future when there is only a single Analytics module, this datapoint will still exist and the unset will be removed.